bindings/go/scip/symbol_formatter.go

```diff
@@ -13,6 +13,15 @@ type SymbolFormatter struct {
 	IncludeDescriptor     func(descriptor string) bool
 }
 
+// VerboseSymbolFormatter formats all parts of the symbol.
+var VerboseSymbolFormatter = SymbolFormatter{
+	IncludeScheme:         func(scheme string) bool { return true },
+	IncludePackageManager: func(_unused string) bool { return true },
```

Comment:
can't these just be `_` -- I'm not sure why you're calling them `unused` when they all are not used in this formatter.

Reply:
Good point. I renamed  these into `_`